lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -1,51 +1,41 @@
 {-# LANGUAGE BangPatterns #-}
-{-# LANGUAGE BlockArguments #-}
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GADTs #-}
-{-# LANGUAGE LambdaCase #-}
-{-# LANGUAGE MultiWayIf #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedLabels #-}
-{-# LANGUAGE RankNTypes #-}
-{-# LANGUAGE ScopedTypeVariables #-}
 
 -- |
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
--- Here we find the "business logic" to manage a Cardano wallet. This is a
--- direct implementation of the model from the [Formal Specification for a Cardano Wallet](https://github.com/input-output-hk/cardano-wallet/blob/master/specifications/wallet/formal-specification-for-a-cardano-wallet.pdf)
--- Note that, this module is purposedly agnostic to how blocks are retrieved or
--- how various types are serialized.
+-- This module implements the "business logic" to manage a Cardano wallet.
+-- It is a direct implementation of the model from the
```

Comment:
```suggestion
-- It is a direct implementation of the model, with extensions, from the
```

Reply:
you know, in the paper ^^^ there was not notion of for example, withdrawal, defined or even mentioned ;-)